scripts/catalog/index.rst

```diff
@@ -0,0 +1,10 @@
+Catalog for GluonNLP
```

Comment:
Can we put everything together in 1 page instead of separate ones?

Reply:
OK, I will make corresponding changes